src/os/mixin/imagemixin.js

```diff
@@ -0,0 +1,23 @@
+goog.provide('os.mixin.Image');
```

Comment:
Please implement all new files using the `goog.module` format. We will have to transform all `goog.provide` files eventually, so we would prefer all new files start in this format.